src/components/util.js

```diff
@@ -34,6 +34,8 @@ module.exports = {
 
         if (actual === type) return
         if (actual.prototype instanceof type) return
+        if (actual.WrappedComponent &&
```

Comment:
@inukshuk this makes `only` work with instances of `InjectIntl(Whatever)`. ok for you?

Reply:
Yes, that was on one of my TODO.